orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -226,14 +239,12 @@ def backward(ctx, *grad_outputs):
                         elif not grad_output.is_contiguous():
                             grad_output = grad_output.contiguous()
                         contiguous_grad_outputs.append(grad_output)
-                    backward_grad_output_ortvalue = [_ortvalue_from_torch_tensor(
-                        grad_output) for grad_output in contiguous_grad_outputs]
 
                     # Run and get results
                     run_id = ctx.run_info.run_id
-                    training_io_binding = ctx.run_info.io_binding
-                    self._training_session.run_backward(backward_grad_output_ortvalue, run_id)
-                    backward_outputs = training_io_binding.get_outputs()
+                    _create_backward_iobinding(ctx.run_info.backward_io_binding, contiguous_grad_outputs, self._onnx_training, self._device, self._onnx_graphs_info.ort_break_op_output_names)
+                    self._training_session.run_backward(ctx.run_info.backward_io_binding, np.int64(run_id))
```

Comment:
>np.int64 [](start = 90, length = 8)

hmm... why need np.int64 here? I need to read more

Reply:
I think by default it passes as int and I get type error.

---
In reply to: [602567079](https://github.com/microsoft/onnxruntime/pull/7077#discussion_r602567079) [](ancestors = 602567079)